contrib/src/main/java/gov/nasa/jpl/aerie/contrib/streamline/debugging/Naming.java

```diff
@@ -66,9 +72,14 @@ public static <T> T name(T thing, String nameFormat, Object... args) {
    * returns empty.
    */
   public static Optional<String> getName(Object thing) {
-    return getName(thing, new NamingContext(null));
+    return getName(thing, new NamingContext());
   }
 
+  // TODO - build a version of getName, or make this the default behavior, (maybe if anonymousName is null?)
```

Comment:
@DavidLegg correct me if I'm wrong - it appears to me that this TODO describes the current behavior of NamingContext when anonymousName is null. Should we delete the comment as done?

Reply:
Yeah, good catch. We can just delete that comment.